scenarios/anomaly-detection/build.gradle

```diff
@@ -12,27 +12,23 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.4'
 }
 
-repositories {
-    mavenLocal()
-    jcenter()
-    mavenCentral()
-}
-
 apply plugin: "java"
 apply plugin: "distribution"
 apply plugin: 'application'
 
 sourceCompatibility = "1.8"
 mainClassName = "io.pravega.anomalydetection.ApplicationMain"
-applicationDefaultJvmArgs = ["-Dlog4j.configuration=file:conf/log4j.properties"]
+applicationDefaultJvmArgs = ["-Dlog4j.configuration=conf/log4j.properties"]
 
 archivesBaseName = 'pravega-flink-scenario-anomaly-detection'
 
 dependencies {
-    compile "org.slf4j:slf4j-log4j12:1.7.14"
     compile "io.pravega:pravega-connectors-flink_2.11:${flinkConnectorVersion}"
     compile "org.apache.flink:flink-streaming-java_2.11:${flinkVersion}"
     compile "org.apache.flink:flink-connector-elasticsearch5_2.11:${flinkVersion}"
+
+    compile "ch.qos.logback:logback-classic:1.1.7"
```

Comment:
We ran into some issues before especially with logging framework and flink runtime. Were you able to successfully use the anomaly app uber jar by uploading and running it from the Flink 1.4 cluster?

Reply:
I tested the application in a local Flink cluster as follows:
- Upload the `jar` via the web UI.
- Specify as `--configDir` the absolute path of the configuration directory.
- Setting in the `app.json` file the property `"controlledEnv": false`.

And it worked in my local cluster. Thanks for the comment, I will add a couple of hints in the README to help users with this execution model.